File: SoloLearn Codes/Hovercraft/Hovercraft.cs
using System;


namespace Sololearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int hover = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine(hover < 7 ? "Loss" : (hover == 7 ? "Broke Even" : "Profit"));
        }
    }
}

File: SoloLearn Codes/Hovercraft/Hovercraft.java
import java.util.Scanner;

public class Hovercraft
{
    public static void main(String[] args) {
        int hover = new Scanner(System.in).nextInt();
        System.out.println(hover < 7 ? "Loss" : (hover == 7 ? "Broke Even" : "Profit"));
	}
}